Add baseline CLI arguments (#104)
obi1kenobi/cargo-semver-checks-action: README.md

* [Input options](#input-options)
* [Example scenarios](#example-scenarios)
  * [Use outside root directory of a crate or a workspace](#use-outside-root-directory-of-a-crate-or-a-workspace)

>   * [Crate is not published on crates.io](#crate-is-not-published-on-cratesio)

Reviewer: ```suggestion
  * [Specifying a baseline](#specifying-a-baseline)
```

| `shared-key`         | A cache key that will be used instead of the automatic key based on the name of the GitHub job and values of the inputs `package`, `exclude` and `manifest-path`. Might be provided e.g. to share the cache between the jobs. | |
| `prefix-key`         | Additional prefix of the cache key, can be set to start a new cache manually. | |
| `github-token`       | The `GITHUB_TOKEN` secret used to download precompiled binaries from GitHub API. If not specified, the [automatic GitHub token](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) provided to the workflow will be used. The token may be alternatively passed in an environment variable `GITHUB_TOKEN`. | `${{ github.token }}` |

> | `baseline-version`   | Version from registry to lookup for a baseline. | |

Reviewer: ```suggestion
| `baseline-version`   | Version from registry to look up as a baseline. | |
```

> | `baseline-rev`       | Git revision to lookup for a baseline. | |

Reviewer: ```suggestion
| `baseline-rev`       | Git revision, branch, or tag to use as a baseline. | |
```

    manifest-path: semver/my-workspace/Cargo.toml  # or just semver/my-workspace/
```

### Specifying a baseline

> By default [`cargo-semver-check`](https://github.com/obi1kenobi/cargo-semver-check) uses crates.io to look up the previous version of the crate, which is used as

Reviewer: ```suggestion
By default [`cargo-semver-checks`](https://github.com/obi1kenobi/cargo-semver-checks) uses crates.io to look up the previous version of the crate, which is used as
```

the baseline for semver-checking the current version of the crate. The following inputs can be used to explicitly specify a baseline instead:

Use a specific version from registry as the baseline:
```yaml
- name: Check semver
  uses: obi1kenobi/cargo-semver-checks-action@v2
  with:
    baseline-version: 4.9.1
```

> Use a git revision as the baseline:

Reviewer: ```suggestion
Use a git revision, branch, or tag as the baseline:
```
